recipe/build-libadios.sh

```diff
@@ -66,37 +47,38 @@ if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]]; then
     export CMAKE_ARGS="${CMAKE_ARGS} -DADIOS2_HAVE_ZFP_CUDA_EXITCODE=0"
 fi
 
-
-cmake              \
-    -S ${SRC_DIR}  \
-    -B build       \
-    ${CMAKE_ARGS}  \
+# shellcheck disable=SC2086
+cmake               \
+    $CMAKE_ARGS     \
+    -S "$SRC_DIR"   \
+    -B build        \
+    -GNinja         \
     -DCMAKE_BUILD_TYPE=Release                \
-    -DCMAKE_VERBOSE_MAKEFILE=ON               \
     -DBUILD_SHARED_LIBS=ON                    \
-    -DBUILD_TESTING=${RUN_TESTS}              \
-    -DCMAKE_CXX_STANDARD=${CXX_STANDARD}      \
-    -DCMAKE_CXX_STANDARD_REQUIRED=ON          \
-    -DCMAKE_CXX_EXTENSIONS=${CXX_EXTENSIONS}  \
-    -DADIOS2_USE_Blosc=ON                     \
+    -DBUILD_TESTING="$RUN_TESTS"              \
     -DADIOS2_USE_BZip2=ON                     \
     -DADIOS2_USE_HDF5=ON                      \
-    -DADIOS2_USE_MPI=${USE_MPI}               \
+    -DADIOS2_USE_MPI="$USE_MPI"               \
     -DADIOS2_USE_PNG=ON                       \
     -DADIOS2_USE_Python=ON                    \
     -DADIOS2_USE_ZeroMQ=ON                    \
     -DADIOS2_USE_ZFP=ON                       \
     -DADIOS2_HAVE_ZFP_CUDA=OFF                \
     -DADIOS2_BUILD_EXAMPLES=OFF               \
     -DADIOS2_RUN_INSTALL_TEST=OFF             \
-    -DPython_EXECUTABLE:FILEPATH=${PYTHON}    \
-    -DPython_INCLUDE_DIR=$(${PYTHON} -c "from sysconfig import get_paths as gp; print(gp()['include'])") \
-    -DCMAKE_INSTALL_LIBDIR=lib        \
-    -DCMAKE_INSTALL_PREFIX=${PREFIX}  \
-    -DKWSYS_LFS_WORKS=0               \
-    -DPNG_PNG_INCLUDE_DIR=${PREFIX}
+    -DPython_EXECUTABLE:FILEPATH="$PYTHON"    \
+    -DPython_INCLUDE_DIR="$("$PYTHON" -c "from sysconfig import get_paths as gp; print(gp()['include'])")" \
+    -DCMAKE_INSTALL_LIBDIR=lib       \
+    -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+    -DPNG_PNG_INCLUDE_DIR="$PREFIX"
+
+if [ -z "$CPU_COUNT" ]
+then
+  CPU_COUNT=2
+fi
+
+cmake --build build "-j$CPU_COUNT" -v
```

Comment:
Org version was simpler in terms of quoting:
```suggestion
cmake --build build -j ${CPU_COUNT} -v
```

did you add `-v` as replacement for `-DCMAKE_VERBOSE_MAKEFILE=ON`?

Reply:
I am not too strong with this.

> did you add -v as replacement for -DCMAKE_VERBOSE_MAKEFILE=ON?

Yes, since CMAKE_VERBOSE_MAKEFILE do not affect ninja builds

Reply:
Perfect, `-v` is better!

Let us just remove the unnecessary quotes again :)